card/mod/05_standard/set/self/head.rb

```diff
@@ -94,26 +107,26 @@ def head_javascript
         $('document').ready(function() {
           $('.card-slot').trigger('slotReady');
         })
-      </script>)
+      </script>
+    HTML
   end
 
-
   def google_analytics_head_javascript
-    if ga_key = Card.setting("*google analytics key") #fixme.  escape this?
-      %{
+    if (ga_key = Card.setting("*google analytics key")) # FIXME: escape this?
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.